lib/get-fail-comment.js

```diff
@@ -1,7 +1,7 @@
 const HOME_URL = "https://github.com/semantic-release/semantic-release";
 const FAQ_URL = `${HOME_URL}/blob/caribou/docs/support/FAQ.md`;
```

Comment:
```suggestion
const FAQ_URL = `${HOME_URL}/blob/master/docs/support/FAQ.md`;
```

this is the FAQ link that needs to be updated